AdminUI/src/app/app.component.ts

```diff
@@ -1,17 +1,25 @@
 import { Component } from '@angular/core';
-import { Router } from '@angular/router';
-import { PrimeNGConfig } from 'primeng/api';
+import { SidebarService } from './services/sidebar-service/sidebar.service';
+
 @Component({
     selector: 'app-root',
     templateUrl: './app.component.html',
     styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-    title = 'admin-ui-app';
+    title = 'AdminUI';
+
+    constructor(private sidebarService: SidebarService) {}
 
-    constructor(private primengConfig: PrimeNGConfig) {}
+    closeSidebar() {
+        this.sidebarService.close();
+    }
+
+    showSidebar(): boolean {
+        return this.sidebarService.isOpen();
```

Comment:
This looks wrong. The method is called `showSidebar`, but it only returns whether the sidebar is open or not.